### PR TITLE
app-846: Fix servo not incrementing after user input

### PR DESCRIPTION
**Pull request by micheal-parks**

### CODE REVIEW

File: web/frontend/src/components/servo.vue [hunk 1]
```diff
@@ -23,10 +23,11 @@ const stop = () => {
 
 const move = (amount: number) => {
   const servo = props.rawStatus;
-  const oldAngle = servo.positionDeg ?? 0;
-  const angle = oldAngle + amount;
 
-  console.log(props.name, angle);
+  // @ts-expect-error @TODO Proto is incorrectly typing this. It expects servo.positionDeg
+  const oldAngle = servo.position_deg ?? 0;
```
Comment (micheal-parks) [inline web/frontend/src/components/servo.vue:28]: Presenting: the fix.

**Comment by stevebriskin** [comment]
please bump the version in app after merging

**Review by DTCurrie — APPROVED** [review]
✅

**Review by mrloureed — APPROVED** [review]
LGTM!

**Pull request merged**